### PR TITLE
Add test stubs and change command/subcommand structure

### PR DESCRIPTION
**Pull request by MDr164**
This commit is just minor cleanup. The test stubs are just
dummy files that add reminders to add tests. In addition
the flags are now extended with shortcuts.

Signed-off-by: Marvin Drees <marvin.drees@9elements.com>

### CODE REVIEW

**Comment by MDr164** [comment]
It was proposed to restructure the commands and its subcommands so the shortcuts will likely go away.

File: stmgr.go [hunk 1]
```diff
@@ -15,108 +15,185 @@ import (
...
+	keygenUsage = `SUBCOMMANDS:
+	certificate:
+		Generate certificates for signing OS packages
+		using ED25519 keys.
+
+Use 'stmgr keygen <SUBCOMMAND> -help' for more info.
 `
 
+	buildUsage = `SUBCOMMANDS:
+	Not yet implemented!
```
Comment (sylv-io) [inline stmgr.go]: remove trailing whitespace

File: stmgr.go [hunk 2]
```diff
@@ -15,108 +15,185 @@ import (
...
+		Set of commands to provision a node for system-transparency
+		usage, like creating and writing a host configuration.
 
 	keygen:
-		Generate certificates for signing OS packages
-		using ED25519 keys.
+		Commands to generate different keys and certificates for
+		system-transparency.
+
+	build:
```
Comment (sylv-io) [inline stmgr.go:33]: Should we remove all "not implemented" commands for now?

File: stmgr.go [hunk 3]
```diff
@@ -15,108 +15,185 @@ import (
...
+	create:
 		Create an OS package from the provided operating
 		system files.
 
 	sign:
 		Sign the provided OS package with your private key.
 
-Use stmgr <COMMAND> -help for more info.
+	show:
+		Not yet implemented!
```
Comment (sylv-io) [inline stmgr.go]: same here

File: stmgr.go (hunk 1 above)
Comment (sylv-io) [reply stmgr.go]: or remove it, too :monkey:

File: stmgr.go [hunk 4]
```diff
@@ -15,108 +15,185 @@ import (
...
+
+Use 'stmgr provision <SUBCOMMAND> -help' for more info.
+`
+
+	keygenUsage = `SUBCOMMANDS:
+	certificate:
+		Generate certificates for signing OS packages
+		using ED25519 keys.
+
+Use 'stmgr keygen <SUBCOMMAND> -help' for more info.
```
Comment (sylv-io) [inline stmgr.go:64]: no help for keygen <subcommand>

**Review by sylv-io — CHANGES_REQUESTED** [review]

File: stmgr.go (hunk 4 above)
Comment (MDr164) [reply stmgr.go:64]: Fixed

File: stmgr.go (hunk 3 above)
Comment (MDr164) [reply stmgr.go]: I removed the reference in the usage text but kept the code for them. They'll be implemented soon anyways.

**Review by sylv-io — APPROVED** [review]

**Pull request merged**